.github/workflows/dependencies/dpcpp.sh

```diff
@@ -25,6 +26,13 @@ sudo apt-get install -y --no-install-recommends \
     g++ gfortran    \
     libopenmpi-dev  \
     openmpi-bin     && \
+du -sh /opt/intel/oneapi/
+du -sh /opt/intel/oneapi/*/*
 sudo rm -rf /opt/intel/oneapi/mkl/latest/lib/intel64/libmkl_sycl.a \
+            /opt/intel/oneapi/mkl/latest/lib/intel64/libmkl_core.a \
```

Comment:
Suggestion added by Dunni Aribuki.
Maybe that is symlinked in the `/opt/intel/oneapi/mkl/latest/lib/intel64` below?